doc/rados/operations/crush-map.rst

```diff
@@ -100,14 +99,14 @@ the hook off in the configuration::
 Custom location hooks
 ---------------------
 
-A customize location hook can be used in place of the generic hook for OSD
-daemon placement in the hierarchy.  (On startup, each OSD ensure its position is
+A customized location hook can be used in place of the generic hook for OSD
+daemon placement in the hierarchy(On startup, each OSD ensures its position is
```

Comment:
any reason to remove the period and the following spaces before the parenthesis?